src/test/java/test/asserts/ActivitiesAsserts.java

```diff
@@ -0,0 +1,49 @@
+package test.asserts;
+
+import data.models.activites.*;
+import org.testng.asserts.SoftAssert;
+import test.common.TestBaseActivities;
+
+public class ActivitiesAsserts extends TestBaseActivities {
+
+    public SoftAssert softAssert = new SoftAssert();
+
+    public void assertListOfActivities(GetActivitiesResponse[] getActivitiesResponses) {
+        for (int i = 0; i <getActivitiesResponses.length; i++){
+            softAssert.assertFalse(getActivitiesResponses[i].getTitle().isEmpty(), "Name is not empty");
+        }
+        softAssert.assertAll();
+    }
+
+    public void assertCreatedActivity (CreateActivityResponse createActivityResponse, CreateActivityRequest createActivityRequest) {
+        softAssert.assertEquals(createActivityResponse.getId(), createActivityRequest.getId(), "ID doesn't match");
+        softAssert.assertEquals(createActivityResponse.getTitle(), createActivityRequest.getTitle(), "Title doesn't match");
+        softAssert.assertEquals(createActivityResponse.getDueDate(), createActivityRequest.getDueDate(), "Due date doesn't match");
+        softAssert.assertEquals(createActivityResponse.getCompleted(), createActivityRequest.getCompleted(), "Completed status doesn't match");
+        softAssert.assertAll();
+    }
+
+    public void assertGetById (GetActivityByIdResponse getActivityByIdResponse){
+        softAssert.assertEquals(getActivityByIdResponse.getId(), getActivityByIdResponse.getId(), "ID doesn't match");
+        softAssert.assertEquals(getActivityByIdResponse.getTitle(), getActivityByIdResponse.getTitle(), "Title doesn't match");
+        softAssert.assertEquals(getActivityByIdResponse.getDueDate(), getActivityByIdResponse.getDueDate(), "Due date doesn't match");
+        softAssert.assertEquals(getActivityByIdResponse.getCompleted(), getActivityByIdResponse.getCompleted(), "Completed status doesn't match");
+        softAssert.assertAll();
+    }
+
+    public void assertUpdateActivity (UpdateActivityResponse updateActivityResponse, UpdateActivityRequest updateActivityRequest){
```

Comment:
Takodje mozes da primetis da imas ovde tri potpuno ista asserta, napravi da se to radi sa jednim, kao i za requeste tako i za response